src/components/ExternalLink/ExternalLink.js

```diff
@@ -12,6 +12,7 @@ type Props = {
   color: string,
   hoverColor?: string,
   showUnderline: boolean,
+  display: string,
 };
 
 class ExternalLink extends Component<Props> {
```

Comment:
Ah, can you please add a `defaultProp` for `display: 'inline-block'`? That way we'll be sure it doesn't affect the layout of any other anchors across the app.

Reply:
Does adding the default prop `display: 'inline-block'` mean I should remove the css `display: inline-block;` from the Anchor that I re-added? Would that css now become redundant?

Reply:
> Does adding the default prop display: 'inline-block' mean I should remove the css display: inline-block; on the Anchor that I re-added? Would that css now become redundant?

Ah my bad, I missed that! 

I would indeed remove it from the styled-component CSS. While what you have now does work, I like the idea of having the inline style be the only "voice" when it comes to that property; feels harder to track if you have to realize that it defaults to the value in the CSS when `display` is undefined.

Reply:
PR updated 😄